src/bv-config.js

```diff
@@ -1,12 +1,10 @@
 //
 // Utility Plugin for setting the configuration
 //
-import { setConfig } from './utils/config'
+import { installFactory } from './utils/plugins'
 
 const BVConfigPlugin = {
-  install(Vue, config = {}) {
-    setConfig(config)
-  }
+  install: installFactory({})
```

Comment:
Would suggest to add a default value to `installFactory` itself so we can just call `installFactory()` here.

Reply:
yeah was thinking that...

Reply:
Checkout `utils/plugins.js` for defaults

Reply:
Yeah, `{ components, directives, plugins } = {}` is enough.

Reply:
Done